docker/dbt/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:latest
+
+ENV DBT_VERSION "0.15.2"
+ENV DBT_DIR /opt/dbt-presto
+
+RUN apk add python3 python3-dev postgresql-dev build-base libffi-dev git bash
+RUN pip3 install --upgrade pip
+RUN pip3 install dbt==${DBT_VERSION}
+RUN mkdir ${DBT_DIR} ${DBT_DIR}/dbt
+ADD dbt ${DBT_DIR}/dbt
+ADD setup.py ${DBT_DIR}
+ADD README.md ${DBT_DIR}
+WORKDIR ${DBT_DIR}
+RUN python3 setup.py install
```

Comment:
This is a very minor style thing, but can you add newlines at the end of the files? Github puts a little marker indicating "no newline" so it's much more visible here than locally.

Reply:
Added: https://github.com/fishtown-analytics/dbt-presto/pull/10/commits/3c9c37968197c2a9edd979b35f08f7deb0c59bd2